New Post: Lost in AI Ethics? The Shared Responsibility Model Offers Clarity (#760)
build-on-aws/content: posts/shared_resp_gen_ai/index.md

---
title: "Lost in AI Ethics? The Shared Responsibility Model Offers Clarity"
description: "Generative AI Shared Responsibility Model"
tags:
  - generative-ai
  - ethics
spaces:
  - generative-ai
waves:
  - generative-ai
authorGithubAlias: aws-banjo
authorName: Banjo Obayomi
date: 2023-09-20
---

|ToC|
|---|

In our modern, tech-savvy world, Generative AI models are no longer the stuff of science fiction. They're here, transforming our interactions with machines and producing outputs that often feel uncannily human. Reminiscent of the cloud computing boom years ago, we are all eager to tap into the potential of these models. However, as with every powerful tool, they come with their own set of challenges.

Recall the initial days of cloud computing. The potential was enormous, but so were the concerns about security and responsibility. The landscape feels similar with Generative AI today. Navigating ethical usage is challenging. And when things go awry, where does the accountability lie? This territory remains ambiguous, with stakeholders often questioning their ethical responsibilities.

> But as usual, history provides us with solutions. The [AWS Shared Responsibility Model](https://aws.amazon.com/compliance/shared-responsibility-model/), which once brought clarity to the world of cloud computing, can be our guiding light. Drawing inspiration from it, I suggest a structured model for Generative AI, aiming to define the roles and responsibilities from the core model providers to the end-users. 

Reviewer: This is specifically the shared responsibility model for security and compliance -- you should make that clear, and below where you share the diagram.   There are other shared responsibility models such as the [one for resilience](https://docs.aws.amazon.com/whitepapers/latest/disaster-recovery-workloads-on-aws/shared-responsibility-model-for-resiliency.html)

> But, as history often does, it provides us with solutions. The [AWS Shared Responsibility Model](https://aws.amazon.com/compliance/shared-responsibility-model/), which once brought clarity to the world of cloud computing, can be our guiding light. Drawing inspiration, we present a structured model for Generative AI, aiming to define the roles and responsibilities from the core model providers to the end-users. 

Reviewer 2: ```suggestion
But as usual, history provides us with solutions. The [AWS Shared Responsibility Model](https://aws.amazon.com/compliance/shared-responsibility-model/), which once brought clarity to the world of cloud computing, can be our guiding light. Drawing inspiration from it, I suggest a structured model for Generative AI, aiming to define the roles and responsibilities from the core model providers to the end-users. 
```

### The Good: Potential of Generative AI
The transformative power of Generative AI is evident across various domains, from enhancing productivity to fueling artistic creativity. Its capabilities offer solutions to long-standing challenges and open doors to new possibilities:

* **Productivity:** One [study](https://www.oneusefulthing.org/p/centaurs-and-cyborgs-on-the-jagged) found that consultants using AI finished 12.2% more tasks on average, completed tasks 25.1% more quickly, and produced 40% higher quality results than those without.
* **Art & Design:** AI-powered tools, such as Stable Diffusion, can create intriguing and [unique visual arts](https://www.reddit.com/r/StableDiffusion/comments/16ew9fz/spiral_town_different_approach_to_qr_monster/), pushing the boundaries of creativity.
* **Research & Innovation:** [Drug discovery](https://blogs.nvidia.com/blog/2023/06/27/insilico-medicine-uses-generative-ai-to-accelerate-drug-discovery/) and material science have seen AI models propose new compounds or solutions that might have taken humans years to conceptualize.
![Reddit user Ugleh Spiral Town art](images/spiral.jpg)

### The Bad: Risks of Generative AI

> However, the very power that makes Generative AI transformative also poses risks when misused or misunderstood:

Reviewer 2: I think one of the biggest challenges, which you don't touch on here, is that of ownership. Data ownership is a tricky question BEFORE you introduce model training (who, for instance, owns a conversation?). But add in the fact that art and design models only function by consuming and synthesizing countless human creations (usually without any compensation or acknowledgement), and it gets even more complicated, ethically.

* **Technological Advancements:** As AI technology advances, new capabilities and potential pitfalls will emerge. Staying ahead of these changes and ensuring that ethics evolve in tandem will be paramount.

However, the challenges are not insurmountable. The shared responsibility model offers a structured approach, ensuring that all stakeholders, from model providers to end-users, play their part in navigating these challenges.

### Let’s Build Responsibly
Generative AI is not just a technological marvel; it's a testament to human ingenuity. But as we harness its power, we must also shoulder the responsibility that comes with it. The shared responsibility model isn't just a framework; it's a **call to action**. It reminds us that ethics in AI isn't the sole responsibility of a select few but a collective commitment.
As we march forward into an exciting AI-driven future, let's do so with purpose, responsibility, and a shared vision. A vision where Generative AI not only transforms industries but does so ethically, benefiting humanity as a whole.

> For further reading I recommend you check out the Amazon Science blog on [responsible AI](https://www.amazon.science/blog/responsible-ai-in-the-generative-era?sc_channel=el&sc_campaign=genaiwave&sc_geo=mult&sc_country=mult&sc_outcome=acq&sc_content=shared_resp_gen_ai),[Ethan Mollick Substack](https://www.oneusefulthing.org/) on how to leverage AI, and [Amazon's Generative AI Hub](https://aws.amazon.com/generative-ai/?sc_channel=el&sc_campaign=genaiwave&sc_geo=mult&sc_country=mult&sc_outcome=acq&sc_content=shared_resp_gen_ai).

Reviewer 2: ```suggestion
For further reading I recommend you check out the Amazon Science blog on [responsible AI](https://www.amazon.science/blog/responsible-ai-in-the-generative-era?sc_channel=el&sc_campaign=genaiwave&sc_geo=mult&sc_country=mult&sc_outcome=acq&sc_content=shared_resp_gen_ai),[Ethan Mollick's Substack](https://www.oneusefulthing.org/) on how to leverage AI, and [Amazon's Generative AI Hub](https://aws.amazon.com/generative-ai/?sc_channel=el&sc_campaign=genaiwave&sc_geo=mult&sc_country=mult&sc_outcome=acq&sc_content=shared_resp_gen_ai).
```
